src/main/io/gps.h

```diff
@@ -314,6 +320,7 @@ typedef enum {
 
 extern gpsData_t gpsData;
 extern gpsSolutionData_t gpsSol;
+extern gpsRelativePos_t gpsrelativePos;
```

Comment:
For OSD, it is easier to compute the value when needed. Or maybe store dLat/dLon from GPS_distance_cm_bearing